install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -498,3 +500,16 @@ func exposeVia3Scale(config map[string]string, syndesis *v1alpha1.Syndesis) {
 		}
 	}
 }
+
```

Comment:
Added the setter, though not sure it's useful at all, since I also added the default value in `install.go`, so form my understanding the setter will never actually set anything